modules/galera/detectors-galera.tf

```diff
@@ -0,0 +1,119 @@
+resource "signalfx_detector" "heartbeat" {
+  name      = format("%s %s", local.detector_name_prefix, "MySQL Galera heartbeat")
```

Comment:
I think we should not add an heartbeat for this module while there already is the mysql one and this will lead to duplicated alerts.